### ISSUE TITLE
debug-cluster/monitor-node-health: YAML file contains many empty lines

### ISSUE THREAD

**Issue by dilyanpalauzov** [kubernetes/website#36361]
The highlightet YAML file at https://kubernetes.io/docs/tasks/debug/debug-cluster/monitor-node-health/#using-kubectl contains an empty line between every two lines.

```yaml
apiVersion: apps/v1

   kind: DaemonSet

   metadata:

     name: node-problem-detector-v0.1

     namespace: kube-system

     labels:

```
and so on.

There shall be no empty lines shown in the browser.

**Comment by amolmote** [comment]
This might not be an issue because it is going to still work even here empty lines are provided. I think this empty lines are provided for the sake of simplicity. I have tested this manifest for `DeamonSet` it works fine.

**Comment by dilyanpalauzov** [comment]
In all other places, like https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/, there are no empty lines after each line.

It does work with that many empty lines, but the empty lines have no added value.

**Comment by jihoon-seo** [comment]
Relevant issues/PRs:
- #34447
- #34736
- #36003

**Comment by sftim** [comment]
/triage accepted

A special case of https://github.com/kubernetes/website/issues/36813

**Comment by jihoon-seo** [comment]
Resolved by #36850
/close
